dev.py

```diff
@@ -1198,5 +1202,49 @@ def authors(ctx_obj, revision_args):
         print('Error caught: Incorrect revision start or revision end')
 
 
+@cli.cls_cmd('ci-local')
+class CILocal():
+    """:flashlight: A tool which provides a handy way to run GitHub Actions locally using Docker
+
+    Installation guide :arrow_down:
+    - Install docker desktop/toolkit
+    - Install act
+        * Using homebrew: brew install act
+        * Using bashscript: curl https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
+
+    Options available:
+        * list (lists all the workflows present)
+        * run-job (runs a particular workflow)
+        * reuse (reuse the containers in act to maintain state)
+
+    Examples (verbose logging enabled by default):
+
+    $ python dev.py ci-local list
+    $ python dev.py ci-local run-job test_meson
+    $ python dev.py ci-local reuse test_meson
```

Comment:
Need line breaks for the rich(?) output:

![Screenshot_20230327_183809](https://user-images.githubusercontent.com/3949932/228072722-7d3b55aa-e818-4ea9-8ca3-3f0034de2394.png)

Reply:
<img width="476" alt="Screenshot 2023-03-28 at 7 34 10 AM" src="https://user-images.githubusercontent.com/17350312/228108097-41e1e5e0-564c-4040-ade6-6a99caff7b56.png">

Cannot reproduce on my machine (But) have updated the style fix. Hopefully, that should resolve this behaviour.